cmd/restic/cmd_mount.go

```diff
@@ -90,10 +90,14 @@ func mount(opts MountOptions, gopts GlobalOptions, mountpoint string) error {
 		return err
 	}
 
-	lock, err := lockRepo(repo)
-	defer unlockRepo(lock)
-	if err != nil {
-		return err
+	if gopts.NoLock {
+		Warnf("not locking the repository\n")
```

Comment:
I don't think there's a need to print this warning. A user hopefully still remembers that they passed the `--no-lock` flag. So let's keep this consistent with the other commands which respect `--no-lock` and remove the warning.

Reply:
@MichaelEischer done in commit a6c94ac.

Let me know if this is good for you.